index.html

```diff
@@ -22,9 +22,10 @@
     <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
       <div class="navbar-nav">
         <a class="nav-item nav-link active" id="log-in-nav" href="#">Log In</a>
+        <a class="nav-item nav-link hidden" id="log-out-nav" href="#">Log Out</a>
         <a class="nav-item nav-link" id="sign-up-nav" href="#">Register</a>
         <!-- <a class="nav-item nav-link" href="#">Profile</a> -->
-        <a class="nav-item nav-link" id="glean-req-nav" href="#">Gleaning Request</a>
+        <a class="nav-item nav-link hidden" id="glean-req-nav" href="#">Gleaning Request</a>
         <a class="nav-item nav-link" target="_blank" href="http://endhunger.org/">Society of St. Andrew</a>
```

Comment:
Please make link https instead of http. Http redirects to https by default

Reply:
Sorry, I was clear in the previous comment. http://endhunger.org/ redirects to https://endhunger.org/